Makefile.work

```diff
@@ -348,6 +348,11 @@ ifneq ($(SIGNING_CERT),)
 endif
 endif
 
+# Allow user-defined environmental variables to be imported into the build containers
+ifneq ($(DOCKER_USER_ENV),)
```

Comment:
How about using SONIC_BUILDER_EXTRA_CMDLINE to add the additional options?
https://github.com/sonic-net/sonic-buildimage/blob/7c3a063255c9e0914111ee89b09e2c12c58ec63b/Makefile.work#L304

Reply:
Yeah, it looks like `SONIC_BUILDER_EXTRA_CMDLINE` works for our use case. You can close this PR if you don't think it adds further value